sporadic memory errors during 'lc package'

This happens very infrequently (maybe 1 out of 100 times), `lc package` will throw the following error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x514c05]

goroutine 1 [running]:
panic(0x8c0fe0, 0xc8200120d0)
    /usr/local/go/src/runtime/panic.go:481 +0x3e6
stash0.eng.lancope.local/dev-infrastructure/project-lifecycle/helpers.RemoveContainersOfImage(0xc82026c848, 0x7, 0x0, 0x0)
    /go/src/stash0.eng.lancope.local/dev-infrastructure/project-lifecycle/helpers/docker.go:89 +0x735
stash0.eng.lancope.local/dev-infrastructure/project-lifecycle/command.RunPackage(0xc8201fe000, 0x0, 0x0)
    /go/src/stash0.eng.lancope.local/dev-infrastructure/project-lifecycle/command/package.go:62 +0x796
```

Guessing this related to 82c30e4 and/or c054cde, we should probably just revert that logic since it doesn't work in all cases (i.e., large docker compose files).

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 ## Unreleased
 
+- When releasing, it now checks for an existing tag or branch that is the same as the `--version` argument
+and gives a better explanation of what happened.
+- Fixed a bug that happened in packaging if the verison of Docker had something like `-rc2` at the end.
+- Fixed issue #4, which would cause a panic in rare cases during packaging.
+
 ## v1.6.0
 
 - Added the ability to override the image that a template uses by setting 

--- a/blackbox-test/features/package.feature
+++ b/blackbox-test/features/package.feature
@@ -1,11 +1,11 @@
 # Copyright 2016 Cisco Systems, Inc.
-# 
+#
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
-# 
+#
 # http://www.apache.org/licenses/LICENSE-2.0
-# 
+#
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.

--- a/command/package.go
+++ b/command/package.go
@@ -1,12 +1,12 @@
 /*
  *  Copyright 2016 Cisco Systems, Inc.
- *  
+ *
  *  Licensed under the Apache License, Version 2.0 (the "License");
  *  you may not use this file except in compliance with the License.
  *  You may obtain a copy of the License at
- *  
+ *
  *  http://www.apache.org/licenses/LICENSE-2.0
- *  
+ *
  *  Unless required by applicable law or agreed to in writing, software
  *  distributed under the License is distributed on an "AS IS" BASIS,
  *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.

--- a/helpers/docker.go
+++ b/helpers/docker.go
@@ -85,18 +85,22 @@ func RemoveContainersOfImage(image string) error {
 			// in the api responses, so for now just check all locations where docker
 			// lists the image value.
 			// It seems that client.InspectContainer.Config.Image is the most trustworthy though
+
 			inspection, err := client.InspectContainer(container.ID)
+
 			if err != nil {
 				logrus.Debugf("could not inspect container %q", container.ID)
-			}
-			if container.Image == image || inspection.Image == image || inspection.Config.Image == image {
-				logrus.Debugf("removing container: %s", container.ID)
-				options := docker.RemoveContainerOptions{ID: container.ID, RemoveVolumes: true, Force: true}
-				if err := client.RemoveContainer(options); err != nil {
-					return err
+			} else {
+				if container.Image == image || inspection.Image == image || inspection.Config.Image == image {
+					logrus.Debugf("removing container: %s", container.ID)
+					options := docker.RemoveContainerOptions{ID: container.ID, RemoveVolumes: true, Force: true}
+					if err := client.RemoveContainer(options); err != nil {
+						return err
+					}
 				}
 			}
 		}
+
 		return nil
 	}
 }

